src/service-providers/dataset-level/dataset-level.interface.ts

```diff
@@ -0,0 +1,122 @@
+import { TCondition } from 'service-providers/filtering-regime/filtering-regime.interface'
+import { IBaseDatasetDescriptor } from 'service-providers/vault-level/vault-level.interface'
+
+// dsinfo
+
+export interface IDsInfoArguments {
+  ds: string
+  note?: string
+}
+export interface IDsInfoClass {
+  title: string
+  values: string[]
+}
+
+export interface IDsInfo extends IBaseDatasetDescriptor {
+  meta: object
```

Comment:
Is it possible to more explicitly specify type for `meta`  key? Or may be use `Record<string, unknown>` for it?

Reply:
Got it, I'll fix it